examples/ec2/route.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: ec2.aws.crossplane.io/v1alpha1
+kind: Route
+metadata:
+  name: example
+spec:
+  forProvider:
+    destinationCIDRBlock: 10.10.10.0/24
+    region: eu-central-1
+    routeTableID: rtb-00000000000000001
```

Comment:
can we add here referencer ? 
reference resolver generation got recently merged - https://github.com/crossplane/crossplane-tools#reference-resolvers if you want to give a try

Reply:
Hi @haarchri,
i talked to @AaronME these days, we had some issues with this PR.
We spent a lot of time with this and decided to first merge and add further features at a later point when we have more free time. 
I tried to use the generator for creating the referencers, but it did not work out of the box, so i dont want to loose more time for now, my team really needs this feature. The initial PR was already created months ago.